packages/shared/src/graphql/notifications.ts

```diff
@@ -41,6 +47,33 @@ export interface NotificationsData {
   notifications: Connection<Notification>;
 }
 
+export enum NotificationIcon {
+  DailyDev = 'DailyDev',
+  CommunityPicks = 'CommunityPicks',
+  Comment = 'Comment',
+  Upvote = 'Upvote',
+  Bell = 'Bell',
+}
+
+export const notificationIcon: Record<
+  NotificationIcon,
+  ComponentType<IconProps>
+> = {
+  [NotificationIcon.DailyDev]: DailyIcon,
+  [NotificationIcon.CommunityPicks]: DailyIcon,
```

Comment:
Why this points to `DailyIcon`?

Reply:
Ah my bad. Development stuff. I was supposed to download the asset for community picks and left it.